package.json

```diff
@@ -138,7 +139,8 @@
     "prepublishOnly": "yarn package-prepare",
     "storybook": "start-storybook -p 6006 -s ./.storybook/public",
     "storybook-docs": "start-storybook --docs -p 6006 -s ./.storybook/public",
-    "build-storybook": "build-storybook -c .storybook -o .storybook-static -s ./.storybook/public"
+    "build-storybook": "build-storybook -c .storybook -o .storybook-static -s ./.storybook/public",
+    "chromatic": "CHROMATIC=true npx chromatic --project-token=940abc073970 --exit-zero-on-changes"
```

Comment:
Shouldn't we store token in secrets like described [here](https://www.chromatic.com/docs/github-actions)?

Reply:
Yes, even more in fact style guide repo is public. I will move it to secrets

Reply:
Done